.github/workflows/CI.yml

```diff
@@ -15,23 +15,34 @@ jobs:
     name: >
       Build and test pyvrp using ${{ matrix.compiler }} 
       ${{ matrix.compiler-version }} and Python ${{ matrix.python-version }}
+      in ${{ matrix.precision }} precision mode
     runs-on: ubuntu-latest
     strategy:
       fail-fast: true
       matrix:
-        include:
-          - compiler: gcc
+        compiler: [gcc, clang]
+        compiler-version: ['10', '12', 'latest']
+        python-version: ['3.8', '3.11']
+        precision: [integer, double]
+        exclude:
+          - compiler: clang
             compiler-version: '10'  # minimum gcc version
-            python-version: '3.8'   # minimum Python version
           - compiler: gcc
-            compiler-version: latest
-            python-version: '3.11'
-          - compiler: clang
             compiler-version: '12'  # minimum clang version
```

Comment:
The `minimum <compiler> version` comments are a bit outdated since we exclude versions now. Maybe update to something like ` test only 12 and latest ` for clang and ` test only 10 and latest ` for GCC?

Reply:
This hasn't changed compared to earlier CI runs - we already only ran 10/latest for GCC, and 12/latest for clang.